### PR TITLE
Fix Transverse Mercator documentation typo

### LINKED ISSUE #2059: Inconsistent syntax for projections in documentation
For some of the [Py]GMT projections the syntax is inconsistent between the [PyGMT overview table](https://www.pygmt.org/dev/projections/index.html#projection-table) and the specific documentation page, e.g.:
- [Mercator projection](https://www.pygmt.org/dev/projections/cyl/cyl_mercator.html)
- [Transverse Mercator projection](https://www.pygmt.org/dev/projections/cyl/cyl_transverse_mercator.html)
- [Cylindrical stereographic](https://www.pygmt.org/dev/projections/cyl/cyl_stereographic.html)
- [Equidistant cylindrical](https://www.pygmt.org/dev/projections/cyl/cyl_equidistant.html)

### PR DESCRIPTION
**Pull request by willschlitzer**
Fixes the Transverse Mercator syntax issues in #2059.

Addresses #2059

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: examples/projections/table/README.txt [hunk 1]
```diff
@@ -49,7 +49,7 @@ The below table shows the projection codes for the 31 GMT projections.
      - :doc:`Eckert VI equal area </projections/misc/misc_eckertVI>`
    * - **L**\ |lon0|/|lat0|/|lat1|/|lat2|/\ *width*
      - :doc:`Lambert conic conformal </projections/conic/conic_lambert>`
-   * - **M**\ [|lon0|\ [/|lat0|]/]\ *width*
+   * - **M**\ |lon0|\ [/|lat0|]/]\ *width*
```
Comment (yvonnefroehlich) [inline examples/projections/table/README.txt:52]: For the Mercator (cylindrical) projection the PyGMT table entry is correct, but the documentation has to be updated (please see https://www.pygmt.org/dev/projections/cyl/cyl_mercator.html).
```suggestion
   * - **M**\ [|lon0|\ [/|lat0|]/]\ *width*
```

**Review by seisman — APPROVED** [review]

**Review by michaelgrund — APPROVED** [review]

**Comment by yvonnefroehlich** [comment]
Hm. Do we plan to have separate PRs for each projection? The PR title was renamed to address only the "Transverse Mercator projection". However, there are some more typos to fix, at least the ones listed at https://github.com/GenericMappingTools/pygmt/issues/2059#issue-1342053087 🙂.

File: examples/projections/table/README.txt (hunk 1 above)
Comment (yvonnefroehlich) [reply examples/projections/table/README.txt]: Thanks for the correction @willschlitzer!
Do you plan to update the Mercator (cylindrical) projection documentation in a separate PR? Please see also my comment below regarding other projections.

**Comment by willschlitzer** [comment]
> Hm. Do we plan to have separate PRs for each projection? The PR title was renamed to address only the "Transverse Mercator projection". However, there are some more typos to fix, at least the ones listed at [#2059 (comment)](https://github.com/GenericMappingTools/pygmt/issues/2059#issue-1342053087) slightly_smiling_face.

@yvonnefroehlich  I had originally planned to fix all of the typos, but I misinterpreted your original comment as asking for someone to fix them. Since I think there needs to be a bit more explanation on what PyGMT typos need fixing (discussed [here](https://github.com/GenericMappingTools/pygmt/issues/2059#issuecomment-1356469092)) vs. what are issues with GMT documentation, it might just be easier if you submitted a PR with your desired fixes. Happy to help, but I don't want to duplicate effort.

**Comment by yvonnefroehlich** [comment]
> > Hm. Do we plan to have separate PRs for each projection? The PR title was renamed to address only the "Transverse Mercator projection". However, there are some more typos to fix, at least the ones listed at [#2059 (comment)](https://github.com/GenericMappingTools/pygmt/issues/2059#issue-1342053087) slightly_smiling_face.
> 
> @yvonnefroehlich I had originally planned to fix all of the typos, but I misinterpreted your original comment as asking for someone to fix them. Since I think there needs to be a bit more explanation on what PyGMT typos need fixing (discussed [here](https://github.com/GenericMappingTools/pygmt/issues/2059#issuecomment-1356469092)) vs. what are issues with GMT documentation, it might just be easier if you submitted a PR with your desired fixes. Happy to help, but I don't want to duplicate effort.

No worries @willschlitzer! Firstly, I wanted only to know how to proceed with `good first issue` issues regarding the release :wink:. But it was totally fine that you directly started working on this 🙂.

@GenericMappingTools/pygmt-maintainers and @willschlitzer Should we leave this PR as is and merge it and open a follow up PR for the remaining typo fixes, or do you prefer closing this PR without merging it and opening a new PR including this one typo fix, to have all these typo fixes together in one PR?
I can try working on this on the weekend, but I can not grant for this yet. Maybe we can bump the upstream GMT PR back to after the PyGMT v0.8.0 release?

**Comment by seisman** [comment]
> Should we leave this PR as is and merge it and open a follow up PR for the remaining typo fixes, or do you prefer closing this PR without merging it and opening a new PR including this one typo fix, to have all these typo fixes together in one PR?

Either works. Or you can push more commits directly to this PR.

**Comment by yvonnefroehlich** [comment]
> > Should we leave this PR as is and merge it and open a follow up PR for the remaining typo fixes, or do you prefer closing this PR without merging it and opening a new PR including this one typo fix, to have all these typo fixes together in one PR?
> 
> Either works. Or you can push more commits directly to this PR.

I prefere working in an own PR on this and starte doing so in PR #2269.

**Comment by seisman** [comment]
> > > Should we leave this PR as is and merge it and open a follow up PR for the remaining typo fixes, or do you prefer closing this PR without merging it and opening a new PR including this one typo fix, to have all these typo fixes together in one PR?
> > 
> > 
> > Either works. Or you can push more commits directly to this PR.
> 
> I prefere working in an own PR on this and starte doing so in PR #2269.

OK. I think we can close this PR.

**Pull request closed without merging**